### DISCUSSION TITLE
Error 1366: Incorrect string value: - folder with diacritics

### DISCUSSION THREAD

**Discussion by jurifi** [photoprism/photoprism#3676] [Q&A]
Hi, I want to ask. When I start indexing, folder with diacritics (in error message is Vánoce) not add to the library.

Error 1366: Incorrect string value: '\xE1noce/...' for column `photoprism`.`albums`.`album_path` at row 1 (create folder)

Please, How can I repair it?

**Comment by lastzero** [comment]
Maybe the string is not unicode or decomposed unicode?
https://en.m.wikipedia.org/wiki/Unicode_equivalence

**Reply by jurifi** [reply]
How can I recognize it?

The files are stored on W10 via SMB. Full name folder is "2022-12-24 - Vánoce". I know the sollution is remove diacritics from name, but It is complicated, because I have it connected to Veeam and I have named liked that about hundred of folders.

So I tried change atributes in docker-compose.yml to "--character-set-server=utf8mb4 --collation-server=utf8mb4_czech_ci" and It not work.

**Reply by jurifi** [reply]
Anybody has idea how to repair it?

**Reply by graciousgrey** [reply]
Maybe the following can solve your issue:

- https://github.com/photoprism/photoprism/discussions/4182#discussioncomment-9203720
